fixed issue with preview view corner radius

gumenuk39 opened the pull request.

Commit:
fixed issue with preview view corner radius

gizmosachin commented:
Hi @gumenuk39, can you file an issue first with screenshots and some context on what the problem here was?

gumenuk39 commented:
If I use the custom size of the Color Slider. Preview view isn't round because of hardcode the 25 for width or height.
File Color Slider
rows 185 - 188
`case .horizontal where autoresizesSubviews:
				preview.bounds.size = CGSize(width: 25, height: bounds.height + 10)
			case .vertical where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.width + 10, height: 25)`
For example in demo I changed the width of color slider 
from 
`colorSlider.widthAnchor.constraint(equalToConstant: 40)`
to
`colorSlider.widthAnchor.constraint(equalToConstant: 26)`
and the result you can see on screenshot
![IMAGE 2019-09-03 13:19:33](https://user-images.githubusercontent.com/14789735/64165303-7a3afa80-ce4d-11e9-824c-fad2c6b8c967.jpg)

I fixed that lines to
`case .horizontal where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.height + 10, height: bounds.height + 10)
			case .vertical where autoresizesSubviews:
				preview.bounds.size = CGSize(width: bounds.width + 10, height: bounds.width + 10)`
and preview View is always round

gizmosachin commented on ColorSlider.podspec:
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ColorSlider'
-  s.version = '4.4'
+  s.version = '4.5'
```
Can you change this to 4.4.1?

gumenuk39 replied:
fixed

gizmosachin commented on Demo/ColorSliderDemo.xcodeproj/project.pbxproj:
```diff
@@ -326,7 +326,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 9BZNV2TE99;
+				DEVELOPMENT_TEAM = 6J9MZ7E65M;
```
Please revert these changes (or remove the `DEVELOPMENT_TEAM` setting from the pbxproj altogether).

gumenuk39 replied:
fixed

gizmosachin commented on Demo/ColorSliderDemo/DemoView.swift:
```diff
@@ -108,7 +108,7 @@ class DemoView: UIView {
 			colorSlider.centerXAnchor.constraint(equalTo: centerXAnchor),
 			colorSlider.bottomAnchor.constraint(equalTo: centerYAnchor),
-			colorSlider.widthAnchor.constraint(equalToConstant: 15),
+			colorSlider.widthAnchor.constraint(equalToConstant: 40),
```
Please revert this change, the `DemoView`'s `ColorSlider` is intentionally 15 points wide.

gumenuk39 replied:
fixed

Commit:
fixed pod version

Commit:
returned demo view size to 15 points

Commit:
fixed development team

Commit:
fixed issue with preview view

Commit:
update pod version

gumenuk39 commented:
also I found another issue:
touch isn't handling when you tap on preview view and it's position is top/bottom or left/right depends on Color slider orientation. for fixing this issue, I changed padding in hit test
was: 
`let padding = -20`
fix:
`
let padding: CGFloat
switch orientation {
     case .vertical:
         padding = -bounds.width
     case .horizontal:
         padding = -bounds.height
}
`

Commit:
added method for setting color programmatically and updating preview view position

Commit:
make setColor func public

gumenuk39 commented:
@gizmosachin what about my PR? all your issues have been fixed

Commit:
probably fixed preview view center position

Commit:
fixes

Commit:
added hack for founding slider progress

Commit:
fix

Commit:
updated pod version
